apps/site/lib/site_web/templates/fare/retail_sales_locations.html.eex

```diff
@@ -8,10 +8,31 @@
 
       <h2>Find a Store Near You</h2>
       <p></p>
-      <%= form_for @conn, @conn.request_path<>"#address-search-input", [as: :location, method: :get, class: "form"], fn _f -> %>
-        <%= input_location(%InputLocation{id: "address-search-input", name: :location, name_index: :address,
```

Comment:
This was the last invocation of the `input_location` input!

**todo(blocking):** Can we either delete that input (`apps/site/lib/site/components/inputs/input_location`) _or_ replace it and its `component.html.eex` with the implementation that you've created here?

Reply:
removed the unused file(s) for now. Made a ticket to move the tnm search component into a more generic form to be reusable.